### ISSUE TITLE
BOT: url validation errors

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#836]
<p>This issue is auto-generated by the <a href="https://github.com/collinbarrett/FilterLists/tree/master/src/FilterLists.Agent">FilterLists.Agent</a>. It is updated at about 9am UTC daily.</p><p>We rely on the help of the community to ensure that the FilterLists site data remains up-to-date. The URLs listed below have been automatically flagged and may <a href="https://github.com/collinbarrett/FilterLists/tree/master/data">need to be updated</a>. Please consider submitting a PR against this issue updating some or all of the URLs accordingly.</p><p>Thanks for your contributions!</p><h1><a href="https://github.com/collinbarrett/FilterLists/blob/master/data/License.json">License.json</a></h1><ul><li><a href="https://www.mozilla.org/MPL/2.0/">https://www.mozilla.org/MPL/2.0/</a><ul><li>This URL appears to redirect and should maybe be changed to "/en-US/MPL/2.0/".</li></ul></li></ul><h1><a href="https://github.com/collinbarrett/FilterLists/blob/master/data/FilterList.json">FilterList.json</a></h1><ul><li><a href="https://gitcdn.xyz/repo/michaeltrimm/hosts-blocking/master/_hosts.txt">https://gitcdn.xyz/repo/michaeltrimm/hosts-blocking/master/_hosts.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/michaeltrimm/hosts-blocking/953e8bfb3a2e16c495c8079a7a28fa1d04e0c169/_hosts.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/mayve/private-adblock-filters/master/Adblock_List.txt">https://gitcdn.xyz/repo/mayve/private-adblock-filters/master/Adblock_List.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/mayve/private-adblock-filters/b0387b8addedc6c9bdbda04ef2cffeba8f4b0c88/Adblock_List.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/slang1998/List-KR/master/filter.txt">https://gitcdn.xyz/repo/slang1998/List-KR/master/filter.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/slang1998/List-KR/159ee28e12427f1c7e58f2a35abc766f1f2c3520/filter.txt".</li></ul></li><li><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=UNHL2VY85PB8N">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=UNHL2VY85PB8N</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.paypal.com/donate?token=hle2Phz247FF185nW-jg9ewV08beQrKWYOohkySKm5KyBiVo36yHccFsvPLiDaRRzQ55IG&country.x=US&locale.x=US".</li></ul></li><li><a href="https://gitcdn.xyz/repo/taylr/linkedinsanity/master/linkedinsanity.txt">https://gitcdn.xyz/repo/taylr/linkedinsanity/master/linkedinsanity.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/taylr/linkedinsanity/f4890056e47e0f5ba16af16bd28813ee181dbf8f/linkedinsanity.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/hoshsadiq/adblock-nocoin-list/master/nocoin.txt">https://gitcdn.xyz/repo/hoshsadiq/adblock-nocoin-list/master/nocoin.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/hoshsadiq/adblock-nocoin-list/44bf21ce951f445f400e4e96f9517caf3489b801/nocoin.txt".</li></ul></li><li><a href="https://sproutsluckycorner.wordpress.com/2017/11/14/my-work-and-contact-resume/#donations">https://sproutsluckycorner.wordpress.com/2017/11/14/my-work-and-contact-resume/#donations</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://gitcdn.xyz/repo/metaphoricgiraffe/tracking-filters/master/trackingfilters.txt">https://gitcdn.xyz/repo/metaphoricgiraffe/tracking-filters/master/trackingfilters.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/metaphoricgiraffe/tracking-filters/021fd3b7012a3d4cdf84d63237afc08b587760e8/trackingfilters.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/liamja/Prebake/master/quiet.txt">https://gitcdn.xyz/repo/liamja/Prebake/master/quiet.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/liamja/Prebake/b61368db6ee75cecdb42c0ad17034d4885bf1f5e/quiet.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/liamja/Prebake/master/obtrusive.txt">https://gitcdn.xyz/repo/liamja/Prebake/master/obtrusive.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/liamja/Prebake/b61368db6ee75cecdb42c0ad17034d4885bf1f5e/obtrusive.txt".</li></ul></li><li><a href="https://discord.me/polskiefiltry">https://discord.me/polskiefiltry</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://gitcdn.xyz/repo/PolishFiltersTeam/PolishAnnoyanceFilters/master/PPB.txt">https://gitcdn.xyz/repo/PolishFiltersTeam/PolishAnnoyanceFilters/master/PPB.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/PolishFiltersTeam/PolishAnnoyanceFilters/49c9dc087795ff3e502015757c515ea8f9b1ef78/PPB.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/LordBadmintonofYorkshire/Overlay-Blocker/master/blocklist.txt">https://gitcdn.xyz/repo/LordBadmintonofYorkshire/Overlay-Blocker/master/blocklist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/LordBadmintonofYorkshire/Overlay-Blocker/2c5f525a8dc9b909a723634746cb1343c776ee5a/blocklist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/genediazjr/nopelist/master/nopelist.txt">https://gitcdn.xyz/repo/genediazjr/nopelist/master/nopelist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/genediazjr/nopelist/fd1f27a053c7068248df7ff25a4330dbcf39a87b/nopelist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/Rpsl/adblock-leadgenerator-list/master/list/list.txt">https://gitcdn.xyz/repo/Rpsl/adblock-leadgenerator-list/master/list/list.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/Rpsl/adblock-leadgenerator-list/1d77c058d23b0501d4d57d872cea42920d6cdca0/list/list.txt".</li></ul></li><li><a href="https://potato.5ch.net/test/read.cgi/android/1450730522/">https://potato.5ch.net/test/read.cgi/android/1450730522/</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://sites.google.com/site/hosts2ch/ja?attredirects=0">https://sites.google.com/site/hosts2ch/ja?attredirects=0</a><ul><li>This URL appears to redirect and should maybe be changed to "https://dabe01e3-a-62cb3a1a-s-sites.googlegroups.com/site/hosts2ch/ja?attachauth=ANoY7coxzERSk_3OG-jye5FopQAd0966h7OYGClM5SViYM-x-qIgrO0ICc1c5av9Yitarwfeuy2OyS9WYWhfRmkmkQKKJJoaLZK31KDfUNqu5_bVJEVG4LrFJFVUAnoApfhyC7g7EOoK1gm9KwAg3N_L5av_tdXjualNcRysDtcAkiagNGlLtRjAIqzZwPPaedIECubTpjhQ&attredirects=0".</li></ul></li><li><a href="https://gitcdn.xyz/repo/HexxiumCreations/threat-list/gh-pages/hexxiumthreatlist.txt">https://gitcdn.xyz/repo/HexxiumCreations/threat-list/gh-pages/hexxiumthreatlist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/HexxiumCreations/threat-list/b5046d1a6ea0d0e30fbbb3b4c392b650e2174bc6/hexxiumthreatlist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/kargig/greek-adblockplus-filter/master/void-gr-filters.txt">https://gitcdn.xyz/repo/kargig/greek-adblockplus-filter/master/void-gr-filters.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/kargig/greek-adblockplus-filter/1f0029753fac8630c88521b9935cde24238e42ab/void-gr-filters.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/DandelionSprout/adfilt/master/AncientLibrary/gmbkslist%20(Hosts).txt">https://gitcdn.xyz/repo/DandelionSprout/adfilt/master/AncientLibrary/gmbkslist%20(Hosts).txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/DandelionSprout/adfilt/1a56c6dfc7c8aa529d82bfe8da2cdc0c053f15b9/AncientLibrary/gmbkslist (Hosts).txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/gfmaster/adblock-korea-contrib/master/filter.txt">https://gitcdn.xyz/repo/gfmaster/adblock-korea-contrib/master/filter.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/gfmaster/adblock-korea-contrib/8239d9e04aee11d5ca8430d32b2e20fb6a60bfdc/filter.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/PolishFiltersTeam/KADhosts/master/KADhosts.txt">https://gitcdn.xyz/repo/PolishFiltersTeam/KADhosts/master/KADhosts.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/PolishFiltersTeam/KADhosts/efbf5feaf2e5d55f6b92b3e92beebd9d19ed6c23/KADhosts.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/PolishFiltersTeam/KAD/master/KAD.txt">https://gitcdn.xyz/repo/PolishFiltersTeam/KAD/master/KAD.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/PolishFiltersTeam/KAD/452e7657d423a0c918f3566059ae39f48f348fe8/KAD.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/lewisje/jansal/master/adblock/hostslt">https://gitcdn.xyz/repo/lewisje/jansal/master/adblock/hostslt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/lewisje/jansal/704669bca9e62b046db66c38091765835631d945/adblock/hostslt".</li></ul></li><li><a href="https://gitcdn.link/repo/lewisje/jansal/master/adblock/hostslt">https://gitcdn.link/repo/lewisje/jansal/master/adblock/hostslt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/lewisje/jansal/704669bca9e62b046db66c38091765835631d945/adblock/hostslt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/lewisje/jansal/master/adblock/hosts">https://gitcdn.xyz/repo/lewisje/jansal/master/adblock/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/lewisje/jansal/704669bca9e62b046db66c38091765835631d945/adblock/hosts".</li></ul></li><li><a href="https://gitcdn.link/repo/lewisje/jansal/master/adblock/hosts">https://gitcdn.link/repo/lewisje/jansal/master/adblock/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/lewisje/jansal/704669bca9e62b046db66c38091765835631d945/adblock/hosts".</li></ul></li><li><a href="https://www.jabcreations.com/web/adblock-subscription">https://www.jabcreations.com/web/adblock-subscription</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.jabcreations.com/web/adblock-subscription?h=1".</li></ul></li><li><a href="https://gitcdn.xyz/repo/piperun/iploggerfilter/master/filterlist">https://gitcdn.xyz/repo/piperun/iploggerfilter/master/filterlist</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/piperun/iploggerfilter/9f92154733530b1d7c1443d7844011028140c652/filterlist".</li></ul></li><li><a href="https://gitcdn.link/repo/piperun/iploggerfilter/master/filterlist">https://gitcdn.link/repo/piperun/iploggerfilter/master/filterlist</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/piperun/iploggerfilter/9f92154733530b1d7c1443d7844011028140c652/filterlist".</li></ul></li><li><a href="https://gitcdn.xyz/repo/ilyakatz/adblock_filters/master/inpage_popups.txt">https://gitcdn.xyz/repo/ilyakatz/adblock_filters/master/inpage_popups.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/ilyakatz/adblock_filters/81cbe154ffaac4f43c716f1819221af7b56dbce2/inpage_popups.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/Rudloff/adblock-imokwithcookies/master/filters.txt">https://gitcdn.xyz/repo/Rudloff/adblock-imokwithcookies/master/filters.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/Rudloff/adblock-imokwithcookies/b6f14f90b64a8a24cc1d98ec1a0c3d7f9b1366dd/filters.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/hufilter/hufilter/master/hufilter.txt">https://gitcdn.xyz/repo/hufilter/hufilter/master/hufilter.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/hufilter/hufilter/a2e860d78db3cf38d277cd20b7b1dd61edc552d5/hufilter.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling-porn/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling-porn/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/gambling-porn/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/gambling/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/uBlock-user/uBO-Personal-Filters/master/uPF.txt">https://gitcdn.xyz/repo/uBlock-user/uBO-Personal-Filters/master/uPF.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/uBlock-user/uBO-Personal-Filters/51c8356af17bec38414c96203d684c6c90a2b130/uPF.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock_ublock.txt">https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock_ublock.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/MajkiIT/polish-ads-filter/92a42eb40cb7852e737077557d7c8c5b5c8d676a/polish-adblock-filters/adblock_ublock.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/IDKwhattoputhere/uBlock-Filters-Plus/master/uBlock-Filters-Plus.txt">https://gitcdn.xyz/repo/IDKwhattoputhere/uBlock-Filters-Plus/master/uBlock-Filters-Plus.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/IDKwhattoputhere/uBlock-Filters-Plus/5299fd8dc98b16286b1208d32bb855ed3de20b0a/uBlock-Filters-Plus.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/byaka/ublock-antiskimming-list/master/build/data.txt">https://gitcdn.xyz/repo/byaka/ublock-antiskimming-list/master/build/data.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/byaka/ublock-antiskimming-list/37f25362c63a929376e3e07fcdd337c3aeb34fe4/build/data.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/unbreak.txt">https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/unbreak.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/uBlockOrigin/uAssets/fe75a1ddbe1a25f07238d8239fd3e4635a1990fc/filters/unbreak.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/privacy.txt">https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/privacy.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/uBlockOrigin/uAssets/fe75a1ddbe1a25f07238d8239fd3e4635a1990fc/filters/privacy.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/filters.txt">https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/filters.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/uBlockOrigin/uAssets/fe75a1ddbe1a25f07238d8239fd3e4635a1990fc/filters/filters.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/experimental.txt">https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/experimental.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/uBlockOrigin/uAssets/fe75a1ddbe1a25f07238d8239fd3e4635a1990fc/filters/experimental.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/badware.txt">https://gitcdn.xyz/repo/uBlockOrigin/uAssets/master/filters/badware.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/uBlockOrigin/uAssets/fe75a1ddbe1a25f07238d8239fd3e4635a1990fc/filters/badware.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling-porn-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling-porn-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/gambling-porn-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/gambling-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/gambling-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/porn-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/porn-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/porn-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/yous/YousList/master/youslist.txt">https://gitcdn.xyz/repo/yous/YousList/master/youslist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/yous/YousList/6ac8200b61a7baebee78a9ea7fe63f3a59a0dad4/youslist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/wiltteri/subscriptions/master/wiltteri-reborn.txt">https://gitcdn.xyz/repo/wiltteri/subscriptions/master/wiltteri-reborn.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/wiltteri/subscriptions/0c0abc24d1dda1bb8460b75fe79cf768a923df75/wiltteri-reborn.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/wiltteri/subscriptions/master/wiltteri.txt">https://gitcdn.xyz/repo/wiltteri/subscriptions/master/wiltteri.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/wiltteri/subscriptions/0c0abc24d1dda1bb8460b75fe79cf768a923df75/wiltteri.txt".</li></ul></li><li><a href="https://discord.me/yourduskquibbles">https://discord.me/yourduskquibbles</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://gitcdn.xyz/repo/yourduskquibbles/webannoyances/master/ultralist.txt">https://gitcdn.xyz/repo/yourduskquibbles/webannoyances/master/ultralist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/yourduskquibbles/webannoyances/44d6b35184a868af7b4a98bd23423ab14762fd86/ultralist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling-porn-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling-porn-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-gambling-porn-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-porn-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-porn-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-porn-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-gambling-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling-porn/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling-porn/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-gambling-porn/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-porn/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-porn/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-porn/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews-gambling/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews-gambling/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/fakenews/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/fakenews/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/social/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/social/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/social/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/porn/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/alternates/porn/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/alternates/porn/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/thoughtconverge/abf/master/abf.txt">https://gitcdn.xyz/repo/thoughtconverge/abf/master/abf.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/thoughtconverge/abf/380315e53986ff3edd30b359cbb75be872df1b1d/abf.txt".</li></ul></li><li><a href="https://www.adblock.ro/">https://www.adblock.ro/</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://gitcdn.xyz/repo/tcptomato/ROad-Block/master/road-block-filters-light.txt">https://gitcdn.xyz/repo/tcptomato/ROad-Block/master/road-block-filters-light.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/tcptomato/ROad-Block/827f55435bed7d125324eb9d460d88ff43770d31/road-block-filters-light.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/betterwebleon/slovenian-list/master/filters.txt">https://gitcdn.xyz/repo/betterwebleon/slovenian-list/master/filters.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/betterwebleon/slovenian-list/e4f87cd5837ecbe6bea6bf39937da5d72b1645aa/filters.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/StevenBlack/hosts/master/data/StevenBlack/hosts">https://gitcdn.xyz/repo/StevenBlack/hosts/master/data/StevenBlack/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/StevenBlack/hosts/5386cec268756eaeb2348c1bdffbe961a8097898/data/StevenBlack/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/StreamingAds/hosts">https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/StreamingAds/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/StreamingAds/hosts".</li></ul></li><li><a href="https://gitcdn.link/repo/FadeMind/hosts.extras/master/StreamingAds/hosts">https://gitcdn.link/repo/FadeMind/hosts.extras/master/StreamingAds/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/StreamingAds/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/Spam404/lists/master/main-blacklist.txt">https://gitcdn.xyz/repo/Spam404/lists/master/main-blacklist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/Spam404/lists/3b255928034a4b0e0ffca3d95c9e6141e29475ab/main-blacklist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/Spam404/lists/master/adblock-list.txt">https://gitcdn.xyz/repo/Spam404/lists/master/adblock-list.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/Spam404/lists/3b255928034a4b0e0ffca3d95c9e6141e29475ab/adblock-list.txt".</li></ul></li><li><a href="https://kb.adguard.com/general/adguard-ad-filters">https://kb.adguard.com/general/adguard-ad-filters</a><ul><li>This URL appears to redirect and should maybe be changed to "/en/general/adguard-ad-filters".</li></ul></li><li><a href="https://gitcdn.xyz/repo/r4vi/block-the-eu-cookie-shit-list/master/filterlist.txt">https://gitcdn.xyz/repo/r4vi/block-the-eu-cookie-shit-list/master/filterlist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/r4vi/block-the-eu-cookie-shit-list/8e165b93c285f207496c716c095f63e8eddafa39/filterlist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/anarki999/Adblock-List-Archive/master/Better.fyiTrackersBlocklist.txt">https://gitcdn.xyz/repo/anarki999/Adblock-List-Archive/master/Better.fyiTrackersBlocklist.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/anarki999/Adblock-List-Archive/7f0d4eb89b0b90c7f50c081e4c9819ade9f4c054/Better.fyiTrackersBlocklist.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/rebelion76/bankiru_plus_adblock_list/master/bankiru_plus.txt">https://gitcdn.xyz/repo/rebelion76/bankiru_plus_adblock_list/master/bankiru_plus.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/rebelion76/bankiru_plus_adblock_list/5a51f8a49577fd9a7b1516a9cc5d9499844b5ee3/bankiru_plus.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/Yhonay/antipopads/master/popads.txt">https://gitcdn.xyz/repo/Yhonay/antipopads/master/popads.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/Yhonay/antipopads/e5df0460c2e9d341332e8f393b387a27dfb9c489/popads.txt".</li></ul></li><li><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=WMGE9MAGGGMHW">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=WMGE9MAGGGMHW</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.Spam/hosts">https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.Spam/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.Spam/hosts".</li></ul></li><li><a href="https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.Spam/hosts">https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.Spam/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.Spam/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.Risk/hosts">https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.Risk/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.Risk/hosts".</li></ul></li><li><a href="https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.Risk/hosts">https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.Risk/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.Risk/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.Dead/hosts">https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.Dead/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.Dead/hosts".</li></ul></li><li><a href="https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.Dead/hosts">https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.Dead/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.Dead/hosts".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php">https://zeustracker.abuse.ch/blocklist.php</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php".</li></ul></li><li><a href="https://zeustracker.abuse.ch/faq.php">https://zeustracker.abuse.ch/faq.php</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=compromised">https://zeustracker.abuse.ch/blocklist.php?download=compromised</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=compromised".</li></ul></li><li><a href="https://gitcdn.xyz/repo/ABPindo/indonesianadblockrules/master/subscriptions/abpindo.txt">https://gitcdn.xyz/repo/ABPindo/indonesianadblockrules/master/subscriptions/abpindo.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/ABPindo/indonesianadblockrules/d825d5a0244a9b16298396f862f0adf79e5594d2/subscriptions/abpindo.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/gioxx/xfiles/master/facebook.txt">https://gitcdn.xyz/repo/gioxx/xfiles/master/facebook.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/gioxx/xfiles/42a4587fee4e7c0edd3b9c89c5e8af5ab914ba71/facebook.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/gioxx/xfiles/master/siteblock.txt">https://gitcdn.xyz/repo/gioxx/xfiles/master/siteblock.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/gioxx/xfiles/42a4587fee4e7c0edd3b9c89c5e8af5ab914ba71/siteblock.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/gioxx/xfiles/master/filtri.txt">https://gitcdn.xyz/repo/gioxx/xfiles/master/filtri.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/gioxx/xfiles/42a4587fee4e7c0edd3b9c89c5e8af5ab914ba71/filtri.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf_paranoid.txt">https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf_paranoid.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/k2jp/abp-japanese-filters/7be1453648ef0b8b0d3c57eb6d9ef059087a793c/abpjf_paranoid.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf.txt">https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/k2jp/abp-japanese-filters/7be1453648ef0b8b0d3c57eb6d9ef059087a793c/abpjf.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf_element_hiding.txt">https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf_element_hiding.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/k2jp/abp-japanese-filters/7be1453648ef0b8b0d3c57eb6d9ef059087a793c/abpjf_element_hiding.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf_3rd_party_sns.txt">https://gitcdn.xyz/repo/k2jp/abp-japanese-filters/master/abpjf_3rd_party_sns.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/k2jp/abp-japanese-filters/7be1453648ef0b8b0d3c57eb6d9ef059087a793c/abpjf_3rd_party_sns.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/abpvn/abpvn/master/filter/abpvn.txt">https://gitcdn.xyz/repo/abpvn/abpvn/master/filter/abpvn.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/abpvn/abpvn/61fa72b1ca0bfebced150a2e3dfcb51939d6fe19/filter/abpvn.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt">https://gitcdn.xyz/repo/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/reek/anti-adblock-killer/a87431fb62ac8f2264f193514ffaa8171418879b/anti-adblock-killer-filters.txt".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=domainblocklist">https://zeustracker.abuse.ch/blocklist.php?download=domainblocklist</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=domainblocklist".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=baddomains">https://zeustracker.abuse.ch/blocklist.php?download=baddomains</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=baddomains".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=ipblocklist">https://zeustracker.abuse.ch/blocklist.php?download=ipblocklist</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=ipblocklist".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=badips">https://zeustracker.abuse.ch/blocklist.php?download=badips</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=badips".</li></ul></li><li><a href="https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.2o7Net/hosts">https://gitcdn.xyz/repo/FadeMind/hosts.extras/master/add.2o7Net/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.2o7Net/hosts".</li></ul></li><li><a href="https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.2o7Net/hosts">https://gitcdn.link/repo/FadeMind/hosts.extras/master/add.2o7Net/hosts</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.link/cdn/FadeMind/hosts.extras/390303c4c23db712e16710eb99f2f1fa8c1ddf91/add.2o7Net/hosts".</li></ul></li><li><a href="https://gitcdn.xyz/repo/kbinani/adblock-youtube-ads/master/signed.txt">https://gitcdn.xyz/repo/kbinani/adblock-youtube-ads/master/signed.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/kbinani/adblock-youtube-ads/b8c6cf8901109188c7334e8ee82ad352d5174b79/signed.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/adblockpolska/Adblock_PL_List/master/adblock_polska.txt">https://gitcdn.xyz/repo/adblockpolska/Adblock_PL_List/master/adblock_polska.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/adblockpolska/Adblock_PL_List/89874ac79421de1902b0522481f3b385250d4211/adblock_polska.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/adblock_social_filters/adblock_social_list.txt">https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/adblock_social_filters/adblock_social_list.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/MajkiIT/polish-ads-filter/92a42eb40cb7852e737077557d7c8c5b5c8d676a/adblock_social_filters/adblock_social_list.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/SlashArash/adblockfa/master/adblockfa.txt">https://gitcdn.xyz/repo/SlashArash/adblockfa/master/adblockfa.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/SlashArash/adblockfa/cb1145a36acc8cd067beda4b9454565b2fd88bbd/adblockfa.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock.txt">https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/MajkiIT/polish-ads-filter/92a42eb40cb7852e737077557d7c8c5b5c8d676a/polish-adblock-filters/adblock.txt".</li></ul></li><li><a href="https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/cookies_filters/adblock_cookies.txt">https://gitcdn.xyz/repo/MajkiIT/polish-ads-filter/master/cookies_filters/adblock_cookies.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/MajkiIT/polish-ads-filter/92a42eb40cb7852e737077557d7c8c5b5c8d676a/cookies_filters/adblock_cookies.txt".</li></ul></li><li><a href="https://sites.google.com/site/logroid/files/hosts.txt?attredirects=0">https://sites.google.com/site/logroid/files/hosts.txt?attredirects=0</a><ul><li>This URL appears to redirect and should maybe be changed to "https://8ec6d0b7-a-62cb3a1a-s-sites.googlegroups.com/site/logroid/files/hosts.txt?attachauth=ANoY7co8F7F9XZ1YR0hkc51Omm0jDgFIgJXlYHPad7wHv-_6jVgAVY6D5Ul24oiBt8xIzL2bGATJF4UvLrPwFdCQMUnmyqff9-T3NE2XOC8Qr52xPc5dV0g9XN8m03BSbk6iawW4iO7uKF2QVpKB1bKtQsFUzk0UrLGjwts2QffxwSXAaBmIxjU6nIO3yFehblk2a_CjFME_ptdvqsuY6N3jWHYayEcB1A%3D%3D&attredirects=0".</li></ul></li><li><a href="https://www.ebates.com/help/troubleshooting.htm#abp">https://www.ebates.com/help/troubleshooting.htm#abp</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.ebates.com/help/category/more-features-115001470107".</li></ul></li><li><a href="https://margevicius.lt/easylistlithuania.txt">https://margevicius.lt/easylistlithuania.txt</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://someonewhocares.org/hosts/">https://someonewhocares.org/hosts/</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://someonewhocares.org/hosts/zero/hosts">https://someonewhocares.org/hosts/zero/hosts</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=RN7R64TJLWQW6">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=RN7R64TJLWQW6</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.paypal.com/donate?token=uD5Ac0jJIh8vpgAnrKehRzQqxxRBuhWVTTNu0QkoCdRllzct5WaJmNj-EMJie0tQSNuMhm&country.x=PL&locale.x=PL".</li></ul></li><li><a href="https://sites.google.com/site/cosmonoteshosts/hosts_for_Windows8.txt">https://sites.google.com/site/cosmonoteshosts/hosts_for_Windows8.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://sites.google.com/site/cosmonoteshosts/hosts_for_Windows8.txt?attredirects=0".</li></ul></li><li><a href="https://sites.google.com/site/cosmonoteshosts/hosts_for_Adaway.txt">https://sites.google.com/site/cosmonoteshosts/hosts_for_Adaway.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://sites.google.com/site/cosmonoteshosts/hosts_for_Adaway.txt?attredirects=0".</li></ul></li><li><a href="https://sites.google.com/site/cosmonoteshosts/hosts_Ultimate.txt">https://sites.google.com/site/cosmonoteshosts/hosts_Ultimate.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://sites.google.com/site/cosmonoteshosts/hosts_Ultimate.txt?attredirects=0".</li></ul></li><li><a href="https://gitlab.com/Kurobeats/phishing_hosts#phishing-site-hosts-generator">https://gitlab.com/Kurobeats/phishing_hosts#phishing-site-hosts-generator</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitlab.com/users/sign_in".</li></ul></li><li><a href="https://www.malekal.com">https://www.malekal.com</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=YMFSMT9MW2Y6Q">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=YMFSMT9MW2Y6Q</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.paypal.com/donate?token=8lNoXgQAvLiu4w7xlishB5PyQw64cSnmZUUEJa-9m6Lt_rGnCPlmqVyKSzg3xscTf7Oumm&country.x=US&locale.x=US".</li></ul></li><li><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=HU4EDJZP2S9QA">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=HU4EDJZP2S9QA</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.paypal.com/donate?token=2850G0YeiwoqeVGRrqrx_WPL_A_GKYuEhp4SaZV7BFgJbPu4KdF_THW6IE-zliS0hdOzLG&country.x=AU&locale.x=AU".</li></ul></li><li><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=CXF2HBWCMSZVL">https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=CXF2HBWCMSZVL</a><ul><li>This URL appears to redirect and should maybe be changed to "https://www.paypal.com/donate?token=Gxji_IxHiGIfLl84cOjf64D5pvxNjrXlOYprmGA6X6ucUFfO0arn6qsTdbFAvXMafeD2-G&country.x=US&locale.x=US".</li></ul></li><li><a href="https://energized.pro/support">https://energized.pro/support</a><ul><li>This URL appears to redirect and should maybe be changed to "http://energized.pro/support/".</li></ul></li><li><a href="https://bads.team/ultrapage#donate">https://bads.team/ultrapage#donate</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://github.com/kowith337/PersonalFilterListCollection/issues?q=label%3A"Site%3A+Gelbooru"">https://github.com/kowith337/PersonalFilterListCollection/issues?q=label%3A"Site%3A+Gelbooru"</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://github.com/kowith337/PersonalFilterListCollection/issues?q=label%3A"Site%3A+APKMirror"">https://github.com/kowith337/PersonalFilterListCollection/issues?q=label%3A"Site%3A+APKMirror"</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://margevicius.lt/">https://margevicius.lt/</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://github.com/kowith337/PersonalFilterListCollection/issues">https://github.com/kowith337/PersonalFilterListCollection/issues</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://github.com/kowith337/PersonalFilterListCollection/blob/master/hosts/readme.md">https://github.com/kowith337/PersonalFilterListCollection/blob/master/hosts/readme.md</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://github.com/kowith337/PersonalFilterListCollection/issues?q=label%3A%22Hosts+File%22">https://github.com/kowith337/PersonalFilterListCollection/issues?q=label%3A%22Hosts+File%22</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://github.com/eEIi0A5L/adblock_filter/blob/master/README.md">https://github.com/eEIi0A5L/adblock_filter/blob/master/README.md</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://gitcdn.xyz/repo/NanoAdblockerLab/NanoContrib/master/dist/placeholder-buster.txt">https://gitcdn.xyz/repo/NanoAdblockerLab/NanoContrib/master/dist/placeholder-buster.txt</a><ul><li>This URL appears to redirect and should maybe be changed to "https://gitcdn.xyz/cdn/NanoAdblockerLab/NanoContrib/b087a76a15c9d1d5c288c6a3a2319b58e2a10d68/dist/placeholder-buster.txt".</li></ul></li><li><a href="http://vivmyccb3jdb7yij.onion/intr0/iVOID.GitLab.io/raw/master/iVOID.hosts">http://vivmyccb3jdb7yij.onion/intr0/iVOID.GitLab.io/raw/master/iVOID.hosts</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="http://vivmyccb3jdb7yij.onion/intr0/DomainVoider/raw/master/DomainVoider.txt">http://vivmyccb3jdb7yij.onion/intr0/DomainVoider/raw/master/DomainVoider.txt</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="http://www.joewein.de/sw/bl-text.htm#donate">http://www.joewein.de/sw/bl-text.htm#donate</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://www.badips.com/documentation#3">https://www.badips.com/documentation#3</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=hostfile">https://zeustracker.abuse.ch/blocklist.php?download=hostfile</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=hostfile".</li></ul></li><li><a href="https://zeustracker.abuse.ch/blocklist.php?download=hostsdeny">https://zeustracker.abuse.ch/blocklist.php?download=hostsdeny</a><ul><li>This URL appears to redirect and should maybe be changed to "https://zeustracker.abuse.ch/byebye.php?download=hostsdeny".</li></ul></li><li><a href="https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/ThirdPartyPrivileged/IDCACnano.txt">https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/ThirdPartyPrivileged/IDCACnano.txt</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li></ul><h1><a href="https://github.com/collinbarrett/FilterLists/blob/master/data/Maintainer.json">Maintainer.json</a></h1><ul><li><a href="https://adguard.com/">https://adguard.com/</a><ul><li>This URL appears to redirect and should maybe be changed to "/en/welcome.html".</li></ul></li><li><a href="https://abpchina.org/forum/">https://abpchina.org/forum/</a><ul><li>This URL appears to redirect and should maybe be changed to "forum.php".</li></ul></li><li><a href="https://margevicius.lt/">https://margevicius.lt/</a><ul><li>This URL might be broken and should maybe be fixed, replaced, or removed.</li></ul></li></ul><h1><a href="https://github.com/collinbarrett/FilterLists/blob/master/data/Software.json">Software.json</a></h1><ul><li><a href="https://adguard.com/">https://adguard.com/</a><ul><li>This URL appears to redirect and should maybe be changed to "/en/welcome.html".</li></ul></li><li><a href="https://addons.mozilla.org/firefox/addon/personal-blocklist/">https://addons.mozilla.org/firefox/addon/personal-blocklist/</a><ul><li>This URL appears to redirect and should maybe be changed to "/en-US/firefox/addon/personal-blocklist/".</li></ul></li><li><a href="https://greasyfork.org/scripts/1682-google-hit-hider-by-domain-search-filter-block-sites">https://greasyfork.org/scripts/1682-google-hit-hider-by-domain-search-filter-block-sites</a><ul><li>This URL appears to redirect and should maybe be changed to "https://greasyfork.org/en/scripts/1682-google-hit-hider-by-domain-search-filter-block-sites".</li></ul></li><li><a href="https://www.samsungknox.com">https://www.samsungknox.com</a><ul><li>This URL appears to redirect and should maybe be changed to "/en".</li></ul></li></ul>

**Comment by DandelionSprout** [comment]
Just a quick note before I begin planning how to fix the rest of the links:

Those links that the agent(?) claims should be redirected to something involving `/en/` or `/en-US/` are false alarms, because most or all such cases are in fact language-adhering redirects that are more preferable to most users than what the agent's suggestions are.

For instance, as someone who uses Norwegian as my main browser language, https://adguard.com/ leads to https://adguard.com/no/welcome.html.

**Comment by DandelionSprout** [comment]
Additionally, all of the agent's suggestions for GitCDN links are also false alarms, because its suggestions are in fact build-specific, whereas the existing links correctly redirect to whatever the newest build at the time is.

**Comment by collinbarrett** [comment]
Awesome, thanks. Yeah, the "Agent" validating URLs is very new, so I'm sure there will be some bugs.

**Comment by DandelionSprout** [comment]
Another bug in the agent that I've come across, is that it doesn't seem to think of links with `#` as being valid, despite how many websites correctly use it to auto-jump to a specific page section.

**Comment by DandelionSprout** [comment]
Would it also be possible to program the agent so that it tries to see if a link is available in HTTPS or not?

**Comment by collinbarrett** [comment]
It's _supposed_ to do that^, but I'm not sure that feature is working. See [here](https://github.com/collinbarrett/FilterLists/blob/69f1072b9d1989951b5d92157ad4c75c7569c607/src/FilterLists.Agent/Features/Urls/ValidateUrls.cs#L116). I'll look into it.

**Comment by DandelionSprout** [comment]
In the event that it does do that after all, then I can only presume that it forgets to check that after a redirection, most notably with links to http://cosmonote.blogspot.jp/ and https://energized.pro/support/.

**Comment by collinbarrett** [comment]
@DandelionSprout I just ran the validation again after #841 , and the list doesn't look much shorter. After some investigation, this seems to be the primary problem: #843 

...I'll work on it.

**Comment by collinbarrett** [comment]
daily updates of this Issue are suspended until #843 is resolved.

**Comment by collinbarrett** [comment]
re-enabled and new run triggered that removed a lot of the false errors on GitHub links in the original post above due to 429 TOO MANY REQUESTS.

**Comment by collinbarrett** [comment]
Closing for now until #940 opens a new Issue.
